lib/reek/ast/node.rb

```diff
@@ -35,52 +35,36 @@ def line
 
       #
       # Carries out a depth-first traversal of this syntax tree, yielding
-      # every Sexp of type `target_type`. The traversal ignores any node
-      # whose type is listed in the Array `ignoring`.
-      # Takes a block as well.
+      # every Sexp of the searched for type or types. The traversal stops at any node
+      # whose type is listed in `ignoring`.
       #
-      # target_type - the type to look for, e.g. :send, :block
-      # ignoring - types to ignore, e.g. [:casgn, :class, :module]
-      # blk - block to execute for every hit. Gets passed in the matching element itself.
+      # If a type is searched for *and*
+      # listed in ignoring, it will be yielded but traversal will not continue
+      # to its children.
```

Comment:
Can we rewrap this comment? The current ragged edge looks like some docs were removed by accident.

```ruby
      # If a type is searched for *and* listed in ignoring, it will
      # be yielded but traversal will not continue to its children.
```

(Look, it wraps to bricktext!)